worker/games.py

```diff
@@ -1297,7 +1297,7 @@ def run_games(
     if "start" in task:
         print("Variable task sizes used. Opening offset = {}".format(opening_offset))
     start_game_index = opening_offset + input_total_games
-    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2**30)
+    run_seed = int(hashlib.sha1(run["_id"].encode("utf-8")).hexdigest(), 16) % (2**64)
```

Comment:
are we sure the result fits a 64bit int, including the sign ?

Reply:
Yes, the SHA1 itself is a 160 bit value modulod to 2^64

Reply:
my point is that the max size of an 64 bit int is  2^63-1 so if we want to fit it, I don't think we can take modulo 2^64

Reply:
> my point is that the max size of an 64 bit int is 2^63-1 so if we want to fit it, I don't think we can take modulo 2^64

The max size of a 64 bit int is 2^64-1, which you can get by modulo 2^64

Reply:
The result of a modulo operation is always positive

Reply:
ok, python modulo is indeed.